ksuid.go

```diff
@@ -49,6 +49,8 @@ var (
 
 	// Represents a completely empty (invalid) KSUID
 	Nil KSUID
+	// Represents the highest value a KSUID can have
```

Comment:
Is this a valid KSUID? If not, should probably be documented.

Reply:
It's valid since it may happen if a KSUID with payload 2^64 is generated on the epoch+2^32 seconds.